www/components/AddCustomSchema.tsx

```diff
@@ -1,101 +1,134 @@
 import { useState } from 'react';
 import { is_valid_custom_schema } from '../lib/schemaValidator';
-import defaultSchema from '../offer_schemas/QmX6CZ7Wf8B79EX5x1PJSityvhtvvFKhkDBCDZK2cd6adF.json';
+import defaultSchema from '../offer_schemas/QmaLinmex9ucfwPWpgSfDPiD1wy7Xy61MzSFLpakgGr7nC.json';
```

Comment:
This is the only real change. But your PR ended up changing the entire file because you have auto formatting. 

This isnt your fault, its a systemic issue with our CI. We need to add better CI to run flakes and styling.. I will add this asap